frontend/src/pages/Settings/Languages/table.tsx

```diff
@@ -194,8 +194,10 @@ interface ItemProps {
 const ItemBadge: FunctionComponent<ItemProps> = ({ cutoff, item }) => {
   const text = useMemo(() => {
     let result = item.language;
-    if (item.hi === "True") {
+    if (item.hi === "only") {
```

Comment:
Do you want to apply the lodash here too? I'm thinking of removing all this let usage in the code side there are not too many and favors immutability